content/docs/networking/load-balancing.md

```diff
@@ -1,47 +1,99 @@
 
-	
-## Network Overlays
-![Overlay Example](/network/Overlay-layers.png)
+## Overlay network
+
+We start by explaining the different network abstraction layers in place and then continue with the semantic addressing.
+
+### Layers
+
+[Overlay Example](/network/Overlay-layers.png)
 
 The abstraction required to enable service-to-service communication spans 3 different layers. 
 
 #### Physical layer
 At the very bottom of the hierarchy, we have the physical layer, where we address machines rather than microservices or containers. 
-In this space, an IP address is an address that can be used to uniquely identify a machine and reach it. In Oakestra, we keep track of this layer, pairing each service with a **Node IP** and **Port**. The pair IP:port enables us to reach multiple devices sharing the same IP address (NAT). Each worker node exposes the network stack required to enable the upper layers of the overlay at the assigned port. 
+In this space, an IP address is an address that can be used to uniquely identify a machine and reach it. In Oakestra, we keep track of this layer, pairing each service with a **Node IP** and **Port**. The pair `IP:Port` enables us to reach multiple devices sharing the same IP address (NAT). Each worker node exposes the network stack required to enable the upper layers of the overlay at the assigned port. 
 
 #### Virtual layer
-Each physical machine is provisioned with a virtual subnetwork. The subnetwork is assigned by the root orchestrator. When instantiating a container, the system provides a so-called **Namespace IP**, which is an IPv4 address provisioned from the virtual subnetwork of the node. This address is used to route the traffic within the platform to the running containers. The current implementation uses a default fixed netmask of 26 bits for the node subnetwork from the private 10.18.0.0/16 network, allowing over 1024 devices, each supporting 64 containers, for a maximum of 65536 containers. Support for more devices and bigger subnetworks can be achieved by changing the default subnetwork to, i.e., 10.0.0.0/8 or even further transitioning to virtual IPv6 networking. 
+Each physical machine is provisioned with a virtual subnetwork. The subnetwork is assigned by the root orchestrator. When instantiating a container, the system provides a so-called **Namespace IP**, which is an address provisioned from the virtual subnetwork of the node. This address is used to route the traffic within the platform to the running containers. The current implementation uses a default fixed netmask of 26 bits for IPv4 [120 bits for IPv6] for the node subnetwork from the private `10.18.0.0/16` [`fc00::/7`] network, allowing over 1024 [a lot of] devices, each supporting 64 [256] containers, for a maximum of 65536 [a lot] containers. Support for more devices and bigger subnetworks can be achieved by changing the default subnetwork to, i.e., 10.0.0.0/8 or even further transitioning to full virtual IPv6 networking. 
```

Comment:
"over 1024 [a lot of] devices" -> over 1000+ devices

"each supporting 64 [256] containers": what does "64 [256]" mean here?

Reply:
the text in brackets refers to the IPv6 implementation -> 64 in IPv4 [256 in IPv6]

Reply:
I guess we should leave it out for now, since we are still IPv4 limited, as there is no standalone IPv6 service deployment